lib/command/src/Obelisk/Command.hs

```diff
@@ -328,9 +330,10 @@ mkObeliskConfig = do
     ObeliskError_ProcessError (ProcessFailure p code) ann ->
       ( "Process exited with code " <> T.pack (show code) <> "; " <> reconstructCommand p
         <> maybe "" ("\n" <>) ann
-      , 2
+      , ExitFailure 2
```

Comment:
Is this 2 a magic number? Can you make an issue to systematize our exit code handling? We should have a semantically meaningful type for exit conditions which get converted to codes in one place.

Reply:
Opened #963